include/audio/audio.hh

```diff
@@ -3,8 +3,6 @@
 
 #include <vector>
 
-struct Audio {
-    std::vector<float> sample;
-};
+using Audio = std::vector<float>;

```

Comment:
Can you change this to a unsigned 16 bit int, since that is what ALSA provides?

Reply:
Have updated that while I'm in there.